soc/arm/atmel_sam0/samd20/dts_fixup.h

```diff
@@ -6,8 +6,6 @@
 
 #define CONFIG_WDT_0_NAME DT_INST_0_ATMEL_SAM0_WATCHDOG_LABEL
 
-#define CONFIG_COUNTER_0_NAME DT_INST_0_ATMEL_SAM0_TC32_LABEL
```

Comment:
Why are you modifying atmel socs with this commit?

Reply:
CONFIG_COUNTER_0_NAME was only being defined in QMSI drivers, nobody else uses this and CI was complaining

Reply:
Will you please note this in the commit message?

Reply:
done